Sources/CorePayments/Networking/HTTPRequest.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+/// :nodoc:
+public struct HTTPRequest {
+    
+    let url: URL
+    let method: HTTPMethod
+    let body: Data?
+    let headers: [HTTPHeader: String]
+    
+    /// :nodoc:
+    public init(
+        url: URL,
+        method: HTTPMethod,
+        body: Data?,
+        headers: [HTTPHeader: String]
```

Comment:
General q, do we have a preference for order in these cases? Since `body` is optional I wonder if it'd read better to be last. I generally prefer required > optional, but don't know that we've ever solidified a style.

Reply:
Yeh, I'm with you - optional last feels nice to me. I updated it to be alphabetical with optionals last in this commit c435d0a5cfafc2626b8768b8f50ba04a3fcd9d54

Reply:
Yeh we could add this to our style guide too

Reply:
Yeah I think it'd be good to style guide! We've been all over the place with it on both BT and PPCP it would be great to have a ⭐ way we wanted to do it

Reply:
This is in [Swift API guidelines](https://www.swift.org/documentation/api-design-guidelines/#parameter-with-defaults-towards-the-end) too!

![Screenshot 2023-08-10 at 3 07 29 PM](https://github.com/paypal/paypal-ios/assets/58225613/06bb6a85-0bd3-480b-95cc-db2826370701)